packages/core/src/lib/toast/toast-container.svelte

```diff
@@ -28,14 +27,13 @@ $: pageIsVisible.set(visibilityState === 'visible');
 <div
   role="status"
   aria-label="Toasts"
-  class="pointer-events-auto fixed bottom-0 left-1/2 top-auto z-max -translate-x-1/2 transform overflow-hidden p-1.5"
```

Comment:
- Remove `pointer-events-auto`, it doesn't do anything
- Remove `overflow-hidden`, it breaks fading when using `animate:flip`